docs/shared/plugins/create-plugin.md

```diff
@@ -0,0 +1,148 @@
+# Create a Plugin
+
+To get started with building a Nx Plugin, run the following command:
+
+```shell
+npx create-nx-plugin my-org --pluginName my-plugin
+```
+
+This command creates a brand-new workspace, and sets up a pre-configured plugin with the specified name.
+
+> Note, the command above will create a plugin with the package name set to `@my-org/my-plugin`. You can pass `--importPath` to provide a different package name.
+
+> If you do not want to create a new workspace, install the `@nrwl/nx-plugin` dependency in an already existing workspace with npm or yarn. Then run `nx g @nrwl/nx-plugin:plugin [pluginName]`.
+
+A new plugin is created with a default generator, executor, and e2e app.
```

Comment:
We are slimming down the default generated plugin to not contain a generator or executor, and e2e is optionally included with `--e2eTestRunner jest` rather than by default.

I imagine more lines in this doc will need to be updated to reflect this